pkg/router/f5/f5.go

```diff
@@ -470,7 +470,7 @@ func (f5 *f5LTM) ensurePolicyExists(policyName string) error {
 		f5.host, policyName)
 
 	err := f5.get(policyUrl, nil)
-	if err != nil && err.(F5Error).httpStatusCode != 404 {
+	if err.(F5Error).httpStatusCode != 404 {
```

Comment:
You aren't supposed to perform type assertions on `nil` values (https://golang.org/ref/spec#Type_assertions).  `error` maybe special (I don't know for sure), but this change still reads incorrectly to me.